README.md

```diff
@@ -52,11 +52,11 @@ jobs:
 
 #### username
 
-The username of the Steam Builder Account that you created in setup step 1.
+The username of the Steam Build Account that you created in setup step 1.
```

Comment:
It used to be called builder account in their documentation. Was this changed?

Reply:
I see it called "Steam Build Account" now: https://partner.steamgames.com/doc/sdk/uploading#Build_Account